pkg/core/interop_system.go

```diff
@@ -221,6 +223,30 @@ func contractIsStandard(ic *interop.Context) error {
 	return nil
 }
 
+// contractCreateMultisigAccount calculates multisig contract scripthash for a
+// given m and a set of public keys.
+func contractCreateMultisigAccount(ic *interop.Context) error {
+	m := ic.VM.Estack().Pop().BigInt()
+	if !m.IsInt64() {
+		return errors.New("m should fit int64")
+	}
+	arr := ic.VM.Estack().Pop().Array()
+	pubs := make(keys.PublicKeys, len(arr))
+	for i, pk := range arr {
+		p, err := keys.NewPublicKeyFromBytes(pk.Value().([]byte), elliptic.P256())
+		if err != nil {
+			return err
+		}
+		pubs[i] = p
+	}
+	script, err := smartcontract.CreateMultiSigRedeemScript(int(m.Int64()), pubs)
```

Comment:
So I can still pass `0xf000000001` and it'll succeed.

Reply:
But why? Isn't `m.IsInt64()` check enough?

Reply:
`int` can be 32-bit.

Reply:
Fixed?